webpack/scenes/Subscriptions/Manifest/SimpleContentAccess.js

```diff
@@ -0,0 +1,103 @@
+import React, { useState, useEffect } from 'react';
+import PropTypes from 'prop-types';
+import { Switch } from '@patternfly/react-core';
+import { OverlayTrigger, Tooltip, Icon } from 'patternfly-react';
+import { translate as __ } from 'foremanReact/common/I18n';
+import { Col, ControlLabel } from 'react-bootstrap';
+
+const SimpleContentAccess = (props) => {
+  const {
+    canToggleSimpleContentAccess,
+    simpleContentAccessValue,
+    enableSimpleContentAccess,
+    disableSimpleContentAccess,
+    setActionInProgress,
+    hideModal,
+  } = props;
+
+  const [isChecked, setChecked] = useState(simpleContentAccessValue);
+
+  useEffect(() => {
+    if (isChecked !== simpleContentAccessValue) {
+      setChecked(simpleContentAccessValue);
+    }
+  }, [simpleContentAccessValue]);
+
+  const toggleSimpleContentAccess = () => {
+    if (simpleContentAccessValue) {
+      disableSimpleContentAccess();
+    } else {
+      enableSimpleContentAccess();
+    }
+  };
+
+  const handleChange = () => {
+    setChecked(!isChecked);
+    setActionInProgress();
+    toggleSimpleContentAccess();
+    hideModal();
+  };
+
+  return (
+    <div id="simple-content-access">
+      <Col sm={4}>
+        <ControlLabel
+          className="control-label"
+          htmlFor="usmaFile"
+          style={{
+            paddingTop: '0',
+            position: 'absolute',
+          }}
+        >
+          <OverlayTrigger
+            overlay={
+              <Tooltip id="sca-tooltip">
+                {__('When Simple Content Access is enabled, hosts can consume from all repositories in their Content View regardless of subscription status.')}
+              </Tooltip>
+            }
+            placement="bottom"
+            trigger={['hover', 'focus']}
+            rootClose={false}
+          >
+            <span>{__('Simple Content Access')}</span>
+          </OverlayTrigger>
+          <OverlayTrigger
+            overlay={
+              <Tooltip id="sca-refresh-tooltip">
+                {__('Toggling Simple Content Access will refresh your manifest.')}
```

Comment:
I still think this particular text should be a regular span, not inside a tooltip.